src/surfaces/actionBar/actionBarSearch.jsx

```diff
@@ -15,6 +23,17 @@ const propTypes = {
     onFocus: PropTypes.func,
     onKeyDown: PropTypes.func,
     placeholder: PropTypes.string,
+    searchWithSelect: PropTypes.shape({
+        dropdownArrowIconType: PropTypes.string,
+        onChange: PropTypes.func,
+        options: PropTypes.arrayOf(PropTypes.shape({})),
+        placeholder: PropTypes.string,
+        value: PropTypes.oneOfType([
+            PropTypes.string,
+            PropTypes.number,
+            PropTypes.shape({}),
+        ]),
+    }),
     style: PropTypes.shape({}), // eslint-disable-line react/forbid-prop-types
```

Comment:
NAB: I think we can remove `// eslint-disable-line react/forbid-prop-types`
I believe that was to suppress the warning if `PropTypes.object` was used instead of `PropTypes.shape({})`.

Reply:
Thank you, Fixed [Commit Link](https://github.com/saddlebackdev/react-cm-ui/pull/452/commits/d5c24a4db77f133280cbb1dad3259ec5cc328f00)